src/js/storage.js

```
@@ -70,7 +70,11 @@ class Storage {
     extend(storage, object);
 
     // Update storage
-    window.localStorage.setItem(this.key, JSON.stringify(storage));
+    try {
+        window.localStorage.setItem(this.key, JSON.stringify(storage));
+    } catch (e) {
+        return;
```

The return here does nothing. It can just be an empty catch.